lib/reporters/lcov-reporter.js

```diff
@@ -0,0 +1,29 @@
+
+var Reporter = require('../reporter');
+
+var lcovRecord = function(data) {
+
+  var str = "";
+  str += 'SF:' + data.fileName + '\n';
```

Comment:
See http://ltp.sourceforge.net/coverage/lcov/geninfo.1.php

There are more data markers that we can add - this is the basics.